v3p/zlib/zconf.h

```diff
@@ -151,7 +145,7 @@
 #    define FAR far
 #  endif
 #endif
-#if defined(__BORLANDC__) && (defined(__SMALL__) || defined(__MEDIUM__))
+#if defined(__MEDIUM__))
```

Comment:
... sorry for confusion. Github apparently takes a double _ as BOLD directive ....